### DISCUSSION TITLE
alcatel_sros not working?

### DISCUSSION THREAD

**Discussion by joaosousa997** [ktbyers/netmiko#2770] [Q&A]
I am trying to redispatch to alcatel_sros tho it does not proceed, while if i use cisco_ios or huawei it is ok.

```
 if "#" in output and "*" not in output and "A:" not in output:
    redispatch(self.vm_conn, device_type='cisco_ios')
elif ">" in output:
    redispatch(self.vm_conn, device_type='huawei')
else:
    redispatch(self.vm_conn, device_type='alcatel_sros')
return output
```

It has no action after trying the redispatch.

**Comment by joaosousa997** [comment]
Model: Nokia SAS-D 6F4T 7210

**Comment by ktbyers** [comment]
You might have to look at the Python debugger.

There is not enough context for me to help you here. For example, what is the value of `output`? What does the Netmiko session_log show. What does the debug log show.

What do you try to do after this?


Regards, Kirk

**Reply by joaosousa997** [reply]
Ah sorry my bad for providing almost no info! I will leave more details later since i am not at work atm.
But i went back to version 3.3.3 (random one i typed) and the alcatel_ssros works now.

Some info i can give rn.

Process is -> Log into a proxy -> log into the router.
From the proxy to the router i check what equipment it is and with that info i redispatch.

Issue being that when i redispatch to the alcatel_ssros it just get's stuck there, while Huawei and cisco_ios redispatch works.

**Reply by ktbyers** [reply]
Sounds good. Let me know when you have some additional information: exception stack trace, debug log, session log, etc.
